schemas/IDLs/org.gel.models.report.avro/4.0.0/CommonRequest.avdl

```diff
@@ -39,7 +39,10 @@ Types:
         BigWig,
         MD5Sum,
         ROH,
-        OTHER
+        OTHER,
+        partition,
+        VFResults,
+        coverage
```

Comment:
@cjohnson12 are these values fixed in this case ?
Is there any documentation for what they are and why they're needed ?

Reply:
All upper case please.
I will get an answer from Illumina on these at the call at 2pm